src/cobra_db/mongo_dao.py

```diff
@@ -197,6 +197,11 @@ def delete_by_id(self, _id: Id):
 
 
 class PatientDao(EntityDao):
+    """
+    Get / insert datainfo in Patient level
```

Comment:
Please do not describe the Patient class here.
The word 'level' should be 'collection'. A good rule of thumb for what should be in the docstring is "Anything that cannot be seen in the signature of the class/method"